index.html

```diff
@@ -0,0 +1,366 @@
+          <div class="name"></div>
+          <div class="title">
+            Screen Readers And Data Tables
+          </div>
+          <div class="content"></div>
+        </div>
+      </div>
+      <div class="authors">
+        <div class="photo rotation"></div>
+        <div class="authors-info-wrapper">
+          <div class="name"></div>
+          <div class="title">
+            Screen Readers And Data Tables
+          </div>
+          <div class="content"></div>
+        </div>
+      </div>
+      <div class="authors">
+        <div class="photo rotation"></div>
+        <div class="authors-info-wrapper">
+          <div class="name"></div>
+          <div class="title">
+            Screen Readers And Data Tables
+          </div>
+          <div class="content"></div>
+        </div>
+      </div>
+      <div class="authors">
+        <div class="photo rotation"></div>
+        <div class="authors-info-wrapper">
+          <div class="name"></div>
+          <div class="title">
+            Screen Readers And Data Tables
+          </div>
+          <div class="content"></div>
+        </div>
+      </div>
+    </div>
+    <div class="more-links"></div>
+
+  </div>
+
+  <div class="design-systems">
+    <div class="before-title"></div>
+    <div class="title"> Design Systems</div>
+    <div class="content"></div>
+    <div class="button"></div>
+    <div class="image rotation"></div>
+  </div>
+
+  <div class="our-guides">
+    <div class="container">
+      <div class="before-title"></div>
+      <div class="title">JavaScript & Frameworks</div>
+      <div class="content"></div>
+      <div class="button"></div>
+    </div>
+    <div class="container">
+      <div class="before-title"></div>
+      <div class="title"> JavaScript & Frameworks</div>
+      <div class="content"></div>
+      <div class="button"></div>
+    </div>
+    <div class="container">
+      <div class="before-title"></div>
+      <div class="title"> JavaScript & Frameworks</div>
+      <div class="content"></div>
+      <div class="button"></div>
+    </div>
+
+    <div class="news-letter">
+      <div class="image"></div>
+      <div class="title">Email Newsletter</div>
+      <div class="content"></div>
+      <div class="input"></div>
+      <div class="bottom-div"></div>
+    </div>
+
+  </div>
+
+
+  <div class="all-guides"></div>
+
+  <div class="footer">
+    <div class="title"></div>
+    <ul class="footer-links">
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
```

Comment:
These footer items should be linked. For instance, we should have something like:
```
<ul>
 <li><a href=""></a></li>
 <li><a href=""></a></li>
</ul>
```
Although content is not important. Yet, this is a requirement.